README.md

```diff
@@ -19,19 +25,19 @@ npm run certificate
 npm run start
 ```
 
-Now you will be able to visit the web-based calendar UI under https://localhost:8080/
-
-The API documentation is available at https://localhost:8080/docs.
+Now you will be able to visit the web-based calendar interface at https://localhost:8080/
 
-### Relaunching the application
+## Relaunching the application
 To relaunch the "das kalender" application, you will need to run the following commands
 ```bash
 cd <your_path>
 npm start
 ```
-<your_path> needs to be replaced with the main directory titled something like *C:\\...\pm_project_rapla*
+<your_path> needs to be replaced with the main directory of the application titled something like *C:\\...\pm_project_rapla*
```

Comment:
Please use linux paths. Windows is ugly... An alternative would be to say: "To relaunch the application, please run `npm start` inside the `pm_project_rapla` folder."